src/xrpld/app/tx/detail/DeleteAccount.cpp

```diff
@@ -51,6 +52,10 @@ DeleteAccount::preflight(PreflightContext const& ctx)
         // An account cannot be deleted and give itself the resulting XRP.
         return temDST_IS_SRC;
```

Comment:
Need to add credentials feature amendment guard here and in other transactors: Escrow, PayChan, Payment for `sfCredentialsIDs`.  
>       if (!enabled and tx.isFieldPresent(sfCredentialsIDs))
>           return temDISABLED.

Reply:
They are added through helper functions `preauthPreflightChack, preauthClaimCheck`